src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Pagination.css';
+
+export const Pagination = (props) => {
+  const { total, page, perPage, onPageChange, withInfo } = props;
+
+  const handleButtonClick = pageNumber => onPageChange(pageNumber);
+
+  const renderButtons = () => (
+    new Array(Math.ceil(total / perPage))
+      .fill('')
+      .map((it, index) => {
+        const buttonClassName = (index + 1 === page)
+          ? 'ui red basic button'
+          : 'ui green basic button';
+
+        const key = index + 1;
```

Comment:
this is almost the same as if you just leave index

Reply:
Used uuid library for key generation instead of index.